### ISSUE TITLE
@emoji-mart/react node_modules package size is 260mb

### ISSUE THREAD

**Issue by gkiely** [missive/emoji-mart#766]
<img width="587" alt="image" src="https://user-images.githubusercontent.com/1948935/210124255-2264f60b-1b83-4fae-86c7-65172a559774.png">

https://cdn.jsdelivr.net/npm/@emoji-mart/react

Reproduction steps:
```
npm install --save emoji-mart @emoji-mart/data @emoji-mart/react
```

OR
```
npm i @emoji-mart/react
```

**Comment by EtienneLem** [comment]
Thanks for the heads up! Should be resolved in **@emoji-mart/react v1.1.1**

**Comment by gkiely** [comment]
👏
